client/src/components/Workflow/Editor/Forms/FormSection.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <Form :id="id" :inputs="inputs" ref="form" @onChange="onChange" />
```

Comment:
If you are forwarding events up the tree you can reduce a lot of manual work by doing this:

```html
<Form v-on="$listeners" />
```
I do this when I want to transparently customize components. You can add a little bit of new functionality or default values, but most of its behavior stays exactly the same as the underlying component you're forwarding the listeners to.

Also, please see my other comment about the problematic "Form" component naming terminology.